tincr/io/design/rapidsmith.tcl

```diff
@@ -0,0 +1,238 @@
+package provide tincr.io.design 0.0
+package require Tcl 8.5
+package require tincr.cad.design 0.0
+package require tincr.cad.device 0.0
+package require tincr.cad.util 0.0
+
+namespace eval ::tincr:: {
+    namespace export \
+        run_rapidsmith_command
+}
+
+## Executes a command send from a running instance of RapidSmith. The "rs_start" and "rs_end"
+#   are used to communicate with RapidSmith when the command has started or completed.
+#
+proc ::tincr::run_rapidsmith_command { cmd } {
+    puts "rs_start"
+    {*}$cmd
```

Comment:
According to the [eval man page](http://wiki.tcl.tk/1017) for a single command, argument expansion is the preferred method.